articles/libraries/auth0-php/management-api.md

```diff
@@ -55,7 +55,7 @@ The `Management` class stores access to endpoints as properties of its instances
 This endpoint is documented [here](/api/management/v2#!/Users/get_users).
 
 ```php
-$results = $mgmt_api->users->search([
+$results = $mgmt_api->users()->getAll([
```

Comment:
@joshcanhelp can you bless the changes on *this* file, please?

Reply:
✅